pkgs/tools/security/clamav-unofficial-sigs.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv, fetchFromGitHub, makeWrapper, clamav, rsync, coreutils, gnutar, gnugrep, curl, gzip, dnsutils, gnupg, glibc, gawk, diffutils }:
+
+stdenv.mkDerivation rec {
+  pname = "clamav-unofficial-sigs";
+  version = "7.2.1";
+
+  src = fetchFromGitHub {
+    owner = "extremeshok";
+    repo = "clamav-unofficial-sigs";
+    rev = version;
+    sha256 = "sha256-utGr7phrLyZojTVg1V2MnlhYmCBP6K6L4OYfug+/5D0=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/bin $out/etc/clamav-unofficial-sigs
+
+    cp clamav-unofficial-sigs.sh $out/bin/clamav-unofficial-sigs
+    chmod +x $out/bin/clamav-unofficial-sigs
+
+    cp config/master.conf $out/etc/clamav-unofficial-sigs
```

Comment:
Is it possible to log to `stdout` instead of [files](https://github.com/extremeshok/clamav-unofficial-sigs/blob/b94125bb7f788dd3f5033f88758023998ade79a6/config/master.conf#L62)? That would be the ideal solution.

If there is no way to log to `stdout` then we should add `LogsDirectory = "clamav-unofficial-sigs";` to the unit as well.